Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -21,7 +21,7 @@ else {
 
 function Get-GraphApiDirectoryReadPermissions() {
     try {
-        $aad = @((Get-AzureADServicePrincipal | Where-Object {$_.ServicePrincipalNames.Contains("https://graph.microsoft.com")}))[0]
+        $aad = @((Get-AzureADServicePrincipal -Filter "ServicePrincipalNames eq 'https://graph.microsoft.com'"))[0]
```

Comment:
The HC tenant had a lot of applications registered. The results where getting paged and the graph api was not being found on that first page.  This filter should only select the graph api application instead of grabbing all applications and then searching for it in the response.